ocaml/testsuite/tests/typing-immediate/immediate.ml

```diff
@@ -1,7 +1,6 @@
 (* TEST
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are unreviewed *)
```

Comment:
* Line 149: I would expect to see a message about the `[@@immediate]` attribute, not just a general blame on the definition of `t`. (Same problem on line 192 and 250.) In contrast, the blame on the definition is fine on line 216 because the definition is indeed well-typed.
* Line 177: Why the `/2`?
* Line 221: There is no test of the error produced here.

The rest of this file has been checked.
*

Reply:
See #2144